app/views/users/_ethical_guidelines_link_or_checklist.html.haml

```diff
@@ -3,20 +3,25 @@
   #  user - the user the membership guidelines checklist belongs to
   #  find_or_create_method - method used to find or create the membership guidelines checklist
   #    defaults to :find_or_create_membership_guidelines_list_for
-- find_or_create = local_assigns.fetch(:find_or_create_method, :find_or_create_membership_guidelines_list_for)
-- if UserChecklistManager.completed_membership_guidelines_checklist?(user)
-  %p
-    = complete_check_icon(html_options: {class: 'is-complete'})
-    = t('users.ethical_guidelines_link_or_checklist.agreed_to', date: UserChecklistManager.membership_guidelines_list_for(user)&.date_completed.to_date.to_s) + ' '
-    = link_to  'https://sverigeshundforetagare.se/medlemsatagande/', target: '_blank' do
-      = t('users.ethical_guidelines_link_or_checklist.membership_guidelines')
-      %sup= external_link_icon
+- if user.former_member?
+  -# disable the Ethical Guidelines checklist button
+  = link_to t('users.ethical_guidelines_link_or_checklist.agree_to_guidelines'), '#', class: 'disabled btn btn-sm btn-primary'
```

Comment:
Shouldn't we put an additional message here (or somewhere on the page) explaining the situation to the user, asking them to contact the membership (email link), .... ?

Reply:
Makes good sense.  @thesuss ?

Reply:
I was thinking that as a step 2. First just make sure they get less frustrated, then explain why. 
But if its quick - do both

Reply:
@thesuss  - what should the message say?

Reply:
I don't know? 
I'm thinking this could have been done 14 days ago if not :) 
I think I can think about it... saturday =)